web-nfc/nfc_url-manual.https.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Web NFC Test: NFCReader NFCReaderOptions url test</title>
+<link rel="author" title="Intel" href="http://www.intel.com"/>
+<link rel="help" href="https://w3c.github.io/web-nfc/"/>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<meta name="flags" content="interact">
+
+<p>Tap an NFC tag to the test device with NFC support.</p>
+
+<p>Note: All the actions need to be done in 60 seconds, otherwise it will get TIMEOUT.</p>
+
+<div id="log"></div>
+
+<script>
+
+"use strict";
+
+setup({ explicit_timeout: true });
+
+promise_test(async t => {
+  const writer = new NFCWriter();
+  const reader1 = new NFCReader({url: `${location.origin}/invalid`});
+  const reader2 = new NFCReader({url: `${location.origin}/custom/path`});
+  await writer.push({
+    url: "/custom/path/update",
+    records: [{ recordType: "text", data: "Hello World" }]
+  });
+
+  const readerWatcher1 = new EventWatcher(t, reader1, ["reading", "error"]);
+  reader1.start();
+  const event1 = await readerWatcher1.wait_for("reading");
+  assert_equals(event1.message.records[0].recordType, "empty");
+
+  const readerWatcher2 = new EventWatcher(t, reader2, ["reading", "error"]);
+  reader2.start();
+  const event2 = await readerWatcher2.wait_for("reading");
+  const record = event2.message.records[0];
+  assert_equals(record.recordType, "text");
+  assert_equals(record.data, "Hello World");
+  assert_equals(event2.message.url, `${location.origin}/custom/path/update`);
+}, "Test that url of NFCReaderOptions is correct.");
```

Comment:
I suppose you are trying to test filtering relevant data sources via URL in `ReaderOptions`. 
So the test name should be something like `NFCReader_options_url-manual.html` and the test description could be more explicit like "Test that the url of NFCReaderOptions filters relevant data sources correctly.".